khelo/lib/ui/flow/team/add_team/add_team_screen.dart

```diff
@@ -93,170 +97,160 @@ class _AddTeamScreenState extends ConsumerState<AddTeamScreen> {
-                        context.l10n.add_team_enter_team_name_placeholder_text,
-                    hintStyle: AppTextStyle.header4
-                        .copyWith(color: context.colorScheme.textDisabled),
-                    suffixIcon: state.checkingForAvailability
-                        ? const AppProgressIndicator(
-                            size: AppProgressIndicatorSize.small,
-                          )
-                        : state.isNameAvailable != null
-                            ? state.isNameAvailable!
-                                ? Icon(
-                                    Icons.check,
-                                    color: context.colorScheme.positive,
-                                  )
-                                : Icon(
-                                    Icons.close,
-                                    color: context.colorScheme.alert,
-                                  )
-                            : null,
-                    border: const OutlineInputBorder(),
-                  ),
-                ),
-                const SizedBox(
-                  height: 16,
-                ),
-                TextField(
-                  onChanged: (value) {
-                    notifier.onValueChange();
-                  },
-                  controller: state.locationController,
-                  style: AppTextStyle.header4
-                      .copyWith(color: context.colorScheme.textPrimary),
-                  decoration: InputDecoration(
-                    hintText: context.l10n.add_team_location_text,
-                    hintStyle: AppTextStyle.header4
-                        .copyWith(color: context.colorScheme.textDisabled),
-                    border: const OutlineInputBorder(),
+                  suffixIcon: state.checkingForAvailability
+                      ? const AppProgressIndicator(
+                          size: AppProgressIndicatorSize.small,
+                        )
+                      : state.isNameAvailable != null
+                          ? state.isNameAvailable!
+                              ? Icon(
+                                  Icons.check,
+                                  color: context.colorScheme.positive,
+                                )
+                              : Icon(
+                                  Icons.close,
+                                  color: context.colorScheme.alert,
+                                )
+                          : null,
+                  hintText:
+                      context.l10n.add_team_enter_team_name_placeholder_text),
+              const SizedBox(height: 8),
+              _textInputField(
+                controller: state.locationController,
+                onChanged: (p0) => notifier.onValueChange(),
+                hintText: context.l10n.add_team_location_text,
+              ),
+              if (widget.editTeam == null) ...[
+                ListTileTheme(
```

Comment:
we might need to add Theme Widget as parent of these as that'll change the colour of checkbox according to colorScheme.